website/docs/r/one_dashboard.html.markdown

```diff
@@ -418,6 +428,4 @@ New Relic dashboards can be imported using their GUID, e.g.
 
 ```bash
 $ terraform import newrelic_one_dashboard.my_dashboard <dashboard GUID>
-```
-
-In addition you can use the [New Relic CLI](https://github.com/newrelic/newrelic-cli#readme) to convert existing dashboards to HCL. [Copy your dashboards as JSON using the UI](https://docs.newrelic.com/docs/query-your-data/explore-query-data/dashboards/dashboards-charts-import-export-data/), save it as a file (for example `terraform.json`), and use the following command to convert it to HCL: `cat terraform.json | newrelic utils terraform dashboard --label my_dashboard_resource`.
```

Comment:
FYI: Discarding this segment, as a more elaborate version of this has been added under "Additional Examples". https://github.com/newrelic/terraform-provider-newrelic/pull/2339/files#diff-56a9d79721e08ffdc1319d625534e335d4514882c278dc01636a374e72a98225R365-R374